readme.md

```diff
@@ -226,6 +226,8 @@ Default: `[]`
 
 This hook enables you to modify the request right before it is sent. Ky will make no further changes to the request after this. The hook function receives normalized input and options as arguments. You could, for example, modify `options.headers` here.
 
+A [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) can be returned from a beforeRequest hook and completely avoid making an HTTP request. This can be used to mock a request, check an internal cache, etc. An **IMPORTANT** consideration when returning a Response from this hook is that, all the following hooks will be skipped, so be **sure to only return a Response from the last one**.
```

Comment:
```suggestion
A [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) instance can be returned from a `beforeRequest` hook to completely avoid making an HTTP request. This is very helpful when mocking a request or using an internal cache.

**Note:** all the following hooks will be skipped.
```

Reply:
This change makes it sound like the following hooks will be skipped, just by using this hook, which is not correct.